.ci/apm-beats-update.groovy

```diff
@@ -35,7 +35,13 @@ pipeline {
       when {
         beforeAgent true
         expression {
-          return isCommentTrigger() || isUserTrigger()
+          def ret = isCommentTrigger() || isUserTrigger()
+          if(!ret){
+            currentBuild.result = 'NOT_BUILT'
+            currentBuild.description = "The build has been skipped"
+            currentBuild.displayName = "#${BUILD_NUMBER}-(Skipped)"
```

Comment:
I wonder if it would be helpful to break out this logic so that the `description` field could include the exact reason that the build was skipped?

Reply:
well, the description should be something not too long to show it in the build list. The reason in both cases is the same, the build does not match the triggers allow, put something like "they build has been skipped due the trigger is a branch scan and the allow ones are manual, and GitHub comment" is long and probably it will generated more questions than answers

Reply:
Makes sense. Could we log it instead? If not, it's not a huge deal. I just think it's nice to have a record of why a skip happened.

Reply:
yep, we can make an echo there